adapter/config/mgwtypes.go

```diff
@@ -125,6 +125,7 @@ type Config struct {
 		ApimCredentials apimCredentials
 		AuthService     authService
 		JwtGenerator    jwtGenerator
+		Cache		    cache
```

Comment:
This looks fine in the Intellij Idea

Reply:
you have both tabs and spaces in here. thats why this has happened shall we stick to one? I think go recommends tab for indentation not spaces

Reply:
Opening the file in vs code, changing something, and pressing "CTRL+S" automatically fixes this.